bin/dotbloat

```diff
@@ -61,7 +61,7 @@ if ! ls -1d --color=auto .[!.]*; then
 fi
 
 # automatic decisions for specific files/directories:
-keeplist='.anthy .local .pki .ssh .zshenv'
+keeplist='.anthy .doom.d .emacs.d .local .pki .ssh .vault-token .zshenv .GA_ClientID .ICAClient'
```

Comment:
This should be removed from the PR. I'm using neither Doom, Emacs, Vault nor Citrix.

Reply:
Dotbloat using config files might be a potential solution to avoid this sort of issues.

I assume that we will never be able to fully get rid of unwanted applications (and their dotfiles in the homedir) in $enterprise environments, unless we start taking a more radical (e.g. containerised) approach.

Reply:
Removed.